src/proc_image_processing/cpu/filters/detectors/pipe_angle_detector.h

```diff
@@ -144,34 +143,36 @@ namespace proc_image_processing {
                 }
 
                 std::sort(objVec.begin(), objVec.end(),
-                    [](ObjectFullData::Ptr a, ObjectFullData::Ptr b) -> bool {
-                        return a->GetArea() > b->GetArea();
-                    });
+                          [](const ObjectFullData::Ptr &a, const ObjectFullData::Ptr &b) -> bool {
+                              return a->getArea() > b->getArea();
```

Comment:
Can we get consistent between files for this (see ln78 of center_coffin_detector.h)